tests/network/bindingplugin.go

```diff
@@ -81,6 +82,8 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 			Expect(err).ToNot(HaveOccurred())
 
 			vmi.Namespace = vm.Namespace
+			Eventually(matcher.ThisVMI(vmi)).Should(matcher.Exist())
```

Comment:
Why do we use a VM and not a VMI in the test?

Reply:
I think at some period in time there was a request to work with the user-facing API, e.g. VM.

This is indeed a border line case, we do too many loops and hoops to make this work with a VM.
But I would prefer we discuss it offline (should we prefer VM or VMI or the opposite).

Reply:
Changed it to create a VMI directly per the review hint.